### ISSUE TITLE
World Requirements 

### ISSUE THREAD

**Issue by TheBambino** [Leader-txt/TShockPlugin-Stellaria-Reset-by-Leader-#1]
Does this plugin require that you must run 4 servers? 

Is the "wrapper" required?

I wish to only run "game1" and "game2".

**Comment by Leader-txt** [comment]
> Does this plugin require that you must run 4 servers?
> 
> Is the "wrapper" required?
> 
> I wish to only run "game1" and "game2".

The "wrapper" isn't required,you can set the "game1" as 'wrapper',can you understand what I mean?

**Comment by TheBambino** [comment]
Yes, I understand. I did as you said said 2 weeks ago. It was very buggy. Will get back to you when I try it out again. Always crashed and the tiles were corrupt. At times carrying over tiles from the other server,

Will post errors and get more detail next time I try it.

**Comment by ghost** [comment]
You fixed it?
